src/NServiceBus.AcceptanceTesting/AcceptanceTestingTransport/AcceptanceTestingTransport.cs

```diff
@@ -1,21 +1,77 @@
-﻿namespace NServiceBus
+﻿using System.Collections.Generic;
```

Comment:
Was flying over the PR out of curiosity

FYI using should go inside the namespace since that is the way we mostly did it here in this repository and in many others. You can add an editor config (`.editorconfig`) to the root folder where you check out all the particular projects. Here is my example. The most important part is `csharp_using_directive_placement = inside_namespace`

```
root = true

[*]
charset=utf-8
insert_final_newline=false
end_of_line=crlf
trim_trailing_whitespace=true
indent_style=space
indent_size=4

csharp_using_directive_placement = inside_namespace
```

Reply:
Note that this will all change with https://github.com/Particular/RepoStandards/pull/29 — I'm not sure it's worth doing anything explicit in this PR regarding this.

Reply:
Yeah sure, I don't really care. Was just pointing out

Reply:
> The most important part is csharp_using_directive_placement = inside_namespace

does that work with code reformat? because currently there is no tooling support that I got working with outside of the R# settings which we killed...

Reply:
It does and is also respected by ReSharper too

Reply:
The main aim of https://github.com/Particular/RepoStandards/pull/29 (and related work in https://github.com/Particular/Particular.CodeRules) is to ensure that tooling aligns with the enforced style settings.